src/cli.rs

```diff
@@ -402,6 +441,7 @@ impl Cli {
             }
             "print_completions" => return Err(print_completions(app, cli.raw_rest_args)),
             "generate_completions" => return Err(generate_completions(app, cli.raw_rest_args)),
+            "generate_manpages" => return Err(generate_manpages(app, cli.raw_rest_args)),
```

Comment:
I am planning to move these subcommands and any dependencies they need (`clap_complete`, `roff`, `chrono`, etc) behind a feature flag.